### PR TITLE
There is problem with repo sync Please fix it buddy

### PR DESCRIPTION
**Pull request by karthikkm007**

### CODE REVIEW

**Comment by Martinusbe** [comment]
Syncing work tree: 100% (613/613), done.

what problem?

**Comment by karthikkm007** [comment]
problem is with this old commit 
i am not able sync work tree , system says fetch error, but only after changing commit now it synced
 i just added **clone-depth="1"** now every thing looks good
6f98f6af62a0326e00e3e648c4c1a577fbb3a236

File: default.xml [hunk 1]
```diff
@@ -414,7 +414,7 @@
   <project path="prebuilts/clang/host/darwin-x86" name="platform/prebuilts/clang/host/darwin-x86" groups="pdk,darwin" clone-depth="1" remote="aosp" />
   <project path="prebuilts/clang/host/linux-x86" name="platform/prebuilts/clang/host/linux-x86" groups="pdk" clone-depth="1" remote="aosp" />
   <project path="prebuilts/devtools" name="platform/prebuilts/devtools" groups="pdk-fs" clone-depth="1" remote="aosp" />
-  <project path="prebuilts/deqp" name="platform/prebuilts/deqp" groups="pdk-fs" remote="aosp" />
```
Comment (Aekkapong18) [inline default.xml:417]: <project path="prebuilts/deqp" name="platform/prebuilts/deqp" groups="pdk-fs" clone-depth="1" remote="aosp" />

**Review by Aekkapong18 — CHANGES_REQUESTED** [review]
<project path="prebuilts/deqp" name="platform/prebuilts/deqp" groups="pdk-fs" clone-depth="1" remote="aosp" />

File: default.xml (hunk 1 above)
Comment (premaca) [reply default.xml:417]: does it happen only with you? There is no sync issues for us, we would have resolved at the earlier anyway if there is any

Comment (karthikkm007) [reply default.xml:417]: It happened with me too
So I have requested a commit with changes

Comment (premaca) [reply default.xml:417]: Thats aosp manifest, i still wonder why is it happening. What command are you using to sync the repo?

**Comment by Mikdadali** [comment]
Validus update every month??

**Comment by Martinusbe** [comment]
yes, every security patch, sometimes random builds from maintainers in between, when stuff got added

**Pull request closed without merging**